[2023-03-03T07:31:09.153+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:31:03.620032+00:00 [queued]>
[2023-03-03T07:31:09.179+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:31:03.620032+00:00 [queued]>
[2023-03-03T07:31:09.181+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:31:09.184+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-03T07:31:09.186+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:31:09.208+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fetch_nexrad_stations> on 2023-03-03 07:31:03.620032+00:00
[2023-03-03T07:31:09.223+0000] {standard_task_runner.py:55} INFO - Started process 1390 to run task
[2023-03-03T07:31:09.237+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'fetch_nexrad_stations', 'manual__2023-03-03T07:31:03.620032+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/scrapper.py', '--cfg-path', '/tmp/tmpqjy4nmb0']
[2023-03-03T07:31:09.240+0000] {standard_task_runner.py:83} INFO - Job 33: Subtask fetch_nexrad_stations
[2023-03-03T07:31:09.344+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:31:03.620032+00:00 [running]> on host 3be715e87c3f
[2023-03-03T07:31:09.475+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=fetch_nexrad_stations
AIRFLOW_CTX_EXECUTION_DATE=2023-03-03T07:31:03.620032+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-03T07:31:03.620032+00:00
[2023-03-03T07:31:09.598+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scrapper.py", line 171, in nexrad_stations
    df = pd.read_fwf(file_path, colspecs=cols, skiprows=[1])
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 762, in read_fwf
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/python_parser.py", line 1203, in __init__
    PythonParser.__init__(self, f, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/python_parser.py", line 96, in __init__
    self._open_handles(f, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/Users/charusingh/Desktop/IS_Courses/Sem5/DAMG7245/Assignment3/nexrad-stations.txt'
[2023-03-03T07:31:09.660+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=fetch_nexrad_stations, execution_date=20230303T073103, start_date=20230303T073109, end_date=20230303T073109
[2023-03-03T07:31:09.689+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 33 for task fetch_nexrad_stations ([Errno 2] No such file or directory: '/Users/charusingh/Desktop/IS_Courses/Sem5/DAMG7245/Assignment3/nexrad-stations.txt'; 1390)
[2023-03-03T07:31:09.771+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-03T07:31:09.899+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
